packages/ics721/src/msg.rs

```diff
@@ -31,6 +31,8 @@ pub struct InstantiateMsg {
     pub pauser: Option<String>,
     /// The admin address for instantiating new cw721 contracts. In case of None, contract is immutable.
     pub cw721_admin: Option<String>,
+    /// The optional contract address length being used for instantiate2. In case of None, default length is 32 (standard in cosmwasm).
+    pub contract_addr_length: Option<usize>,
```

Comment:
usize has different sizes in dev and production. Better use u32 here

Reply:
fixed and changed to u32